packages/edit-site/src/components/global-styles/dimensions-panel.js

```diff
@@ -38,6 +40,13 @@ function useHasMargin( name ) {
 	return settings && supports.includes( 'margin' );
 }
 
+function useHasGap( name ) {
```

Comment:
Can we update this to hide it just for the "blocks" level for now. Meaning return false when `name` is defined I think.

Reply:
That'll do it 👍